test/functional/api/extmark_spec.lua

```diff
@@ -173,6 +173,18 @@ describe('API/extmarks', function()
     })
   end)
 
+  it('end_col is -1', function()
+    set_extmark(ns, marks[1], 0, 0, {
+      end_col = -1,
+      end_row = 0,
+      strict = false,
+    })
+    eq(
+      "Invalid 'end_col': out of range",
+      pcall_err(set_extmark, ns, marks[2], 0, 0, { end_col = -1 })
```

Comment:
what is this supposed to be testing? is `marks[2]` empty or something?

Reply:
From `nvim_buf_set_extmark` docs:
```
- strict: boolean that indicates extmark should not be placed
  if the line or column value is past the end of the
  buffer or end of the line respectively. Defaults to true.
```
So I think the purpose of the test you're referring to is to test what happens when `strict = true`.

I guess the better way of doing this is explicitly setting `strict` to `true`, but I tried to keep it consistent with other tests.

Reply:
> I guess the better way of doing this is explicitly setting `strict` to `true`, but I tried to keep it consistent with other tests.

That would definitely clarify the test. Other tests can be updated if you want.

Reply:
> That would definitely clarify the test. Other tests can be updated if you want.

I'll make sure to add it right away. Also, should I open a separate PR to clarify other tests?